### PR TITLE
chore: add pre-commit

### PR DESCRIPTION
**Pull request by matthieu-foucault**
### Description: 

Python is needed for `pre-commit`, and was missing from the `.tool-versions`

### CODE REVIEW

**Comment by matthieu-foucault** [comment]
Noticing we have a `requirements.txt` file with pre-commit as a dependency, but no pre-commit configuration...

**Review by matthieu-foucault — COMMENTED** [review]
@AntBush I can't actually request changes since I own this PR

File: README.md [hunk 1]
```diff
@@ -120,3 +120,11 @@ We developed some guardrails (i.e. GitHub actions) to:
 
 - ensure that changes that are part of a release are immutable: [immutable-sqitch-change.yml}(.github/workflows/immutable-sqitch-change.yml)
 - ensure that the sqitch plan ends with a tag on the `main` branch, preventing deployments if it is not the case. Our release command automatically sets this tag: [pre-release.yml](.github/workflows/pre-release.yml)
+
+### Pre-Commit Hooks
+
+_The following is to be done in the root directory_
+
+To install the pre-commit hooks, first install the asdf tools using `make install_asdf_tools` also install the python tools using `pip install -r requirements.txt` and `pre-commit install`.
```
[the remark this answers is not in the document]
Comment (AntBush) [reply README.md:128]: So I still need to update the docs, but now this change is causing the CI to fail when attempting to install cocogitto

https://github.com/bcgov/CONN-CCBC-portal/actions/runs/3045639046/jobs/4907440164#step:4:5819

[the remark this answers is not in the document]
Comment (matthieu-foucault) [reply README.md]: Let's remove `install_git_hooks` as a dependency of `install_dev_tools` then. We can run it separately with `make install_git_hooks`

**Review by matthieu-foucault — COMMENTED** [review]
@AntBush Look good to me, feel free to approve and merge (I can't approve, it's my PR)

**Review by AntBush — APPROVED** [review]
LGTMatthieu

**Review by VB-at-Bis — APPROVED** [review]
Looks good

**Comment by VB-at-Bis** [comment]
@AntBush  https://github.com/bcgov/CONN-CCBC-portal/pull/434 can be merged now

**Pull request merged**